cloudprober/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          args: ['--config_file', '/cfg/cloudprober.{{- .Values.configFormat -}}', '--logtostderr']
+          args: ['--config_file', '/cfg/cloudprober.{{- .Values.configFormat -}}', '--logtostderr'{{- include "join-args" .Values.extraArgs -}}]
```

Comment:
Instead of defining and including a template, how about doing something like this, i.e.  merge extraArgs directly here?
https://github.com/grafana/helm-charts/blob/4401adc2780fba7a5a8da3dcda1a2d6fb039bccc/charts/grafana/templates/_pod.tpl#L945